content/participate/document.adoc

```diff
@@ -55,7 +55,7 @@ You can open any GitHub Markdown or Asciidoc page in any Jenkins repo, click _Ed
 Useful links:
 
 * link:https://issues.jenkins-ci.org/issues/?filter=18650&jql=(project%20%3D%20WEBSITE%20or%20labels%20in%20(documentation%2C%20docs-sig))%20AND%20labels%20%3D%20newbie-friendly%20and%20status%20in%20(Open%2C%20Reopened%2C%20%22To%20Do%22)[Newbie-friendly documentation tasks]
```

Comment:
Sorry that I didn't read carefully enough on my first review.  This is the one that should reference friendly issues.
```suggestion
* link:https://github.com/jenkins-infra/jenkins.io/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22[Newbie-friendly documentation tasks]
```